app/(components)/EditTicketForm.jsx

```diff
@@ -194,11 +221,18 @@ const EditTicketForm = ({ ticket }) => {
           <option value="started">Started</option>
           <option value="done">Done</option>
         </select>
-        <input
-          type="submit"
-          className="btn max-w-xs"
-          value={EDITMODE ? "Update Ticket" : "Create Ticket"}
-        />
+        <div className="flex flex-row flex-wrap justify-between gap-2">
+          {/* <input
+            type="submit"
+            className="btn max-w-xs"
+            value={EDITMODE ? "Update Ticket" : "Create Ticket"}
+          /> */}
+          <button type="submit" className="btn max-w-xs">
```

Comment:
this is an unnecessary change, the commented-out section of code:

```javascript
          {/* <input
            type="submit"
            className="btn max-w-xs"
            value={EDITMODE ? "Update Ticket" : "Create Ticket"}
          /> */}
```

is already functional and serves its purpose. Therefore, this change is unnecessary. Please refrain from committing changes that do not contribute to resolving the issue at hand.

Once this is addressed, your PR will be ready for merging.
Thank you

Reply:
Sir actually the thing was that input and button had different styles and their heights were different, hence it was looking odd and not good. 
Therefore I converted input into a button.
Rest as you say 😊